packages/config/__tests__/Config.api.test.ts

```diff
@@ -216,6 +216,35 @@ describe("Config API tests", () => {
                 expect(profile).toBeNull();
             });
         });
+        describe("expandPath", () => {
+            it("should expand a short proeprty path", async () => {
+                const config = await Config.load(MY_APP);
+                const profilePath = "lpar1.zosmf";
+                expect(config.api.profiles.expandPath(profilePath)).toEqual("profiles.lpar1.profiles.zosmf");
+            });
+            it.skip("should not expand a complete path", async () => {
```

Comment:
There are a couple disabled tests that should probably be re-enabled.

Reply:
Since it is possible to have a `profiles.zosmf` profile, we shouldn't assume the given path is "wrong".
Same goes for the `getProfileNameFromPath` method
```bash
zFernand0 @ SSH:~/gh# zowe config list 
profiles: 
  profiles: 
    profiles: 
      zosmf: 
        properties: 
          rejectUnauthorized: false
          host:               example.com
          port:               443
          user:               USER
          password:           PASS
defaults: 
  zosmf: profiles.zosmf
autoStore: true
zFernand0 @ SSH:~/gh# zowe files list ds sys1.parmlib
SYS1.PARMLIB
SYS1.PARMLIB.D270622
SYS1.PARMLIB.POK
SYS1.PARMLIB.SHIP
```